lib/validators/add_locations.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
```

Comment:
How do you feel about changing the name of this file to `location.js` rather than `add_locations.js` to be consistent with the naming of the movie payload validator?